projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.ts

```diff
@@ -394,16 +398,22 @@ export class IgxTreeGridComponent extends IgxGridBaseComponent {
     }
 
     /**
-     * @hidden
+     * Removes the `IgxGridRowComponent` and the corresponding data record by primary key.
+     * Requires that the `primaryKey` property is set.
+     * The method accept rowSelector as a parameter, which is the rowID.
+     * ```typescript
+     * this.grid1.deleteRow(0);
+     * ```
+     * @param rowSelector
+     * @memberof IgxGridBaseComponent
```

Comment:
This shouldn't be exposed, unless required. Samples should use https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/classes/igxtreegridcomponent.html#deleterow instead

Reply:
done!